.travis.yml

```diff
@@ -29,14 +29,12 @@ matrix:
     - language: r
       r: release
       before_install:
-        - Rscript -e 'install.packages("devtools", dependencies = TRUE)'
+        - Rscript -e 'options(repos = c(CRAN = "https://demo.rstudiopm.com/all/__linux__/xenial/latest")); install.packages("devtools")'
```

Comment:
I am not sure this has improved the runtime -- the packages are still compiled from source.

Reply:
You are right. Not sure why it does not work on Travis. Will keep the original. But remove `dependencies = TRUE` did reduce the package sizes to install. So I will keep it.